tests/test_regionaltransit.py

```diff
@@ -16,7 +16,12 @@ def test_name():
 
 
 def test_description():
-    assert parsed_items[0]['description'] == 'Test Description'
+    assert 'Test Description' == ("The RTA Board of Directors is a 16-member group of professionals"
+                        "governing the activities and initiatives of the RTA. The RTA is "
+                        "charged with financial oversight, funding, and regional transit "
+                        "planning for the region’s transit operators: the Chicago Transit
```

Comment:
it looks like a closing quote is missing here as well. Also, Python is sensitive to [indentation](https://stackoverflow.com/questions/13884499/what-is-python-whitespace-and-how-does-it-work).  I'd recommend formatting these quotes to look more like the description in your spider, including the way the quotes are indented (which is essential) and the spacing (which just helps readability)